tests/test_cli.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+from typer.testing import CliRunner
+
+from gtfs.__main__ import app
+from gtfs.utils.constants import *
+from gtfs.utils.geom import bbox_contains_bbox, bbox_intersects_bbox
+
+
+@pytest.fixture(scope="module")
+def runner():
+    return CliRunner()
+
+
+class Test_Geom_Functions:
+    @pytest.mark.parametrize(
+        "bbox1, bbox2",
+        [bbox_covers_bbox, bbox_contained_inside_bbox],
+    )
+    def test_geom_contains(self, bbox1, bbox2):
+        assert bbox_contains_bbox(bbox1, bbox2) == True
+
+    @pytest.mark.parametrize(
+        "bbox1, bbox2",
+        [bbox_outside_bbox, bbox_intersecting_bbox],
+    )
+    def test_geom_contains_not(self, bbox1, bbox2):
+        assert bbox_contains_bbox(bbox1, bbox2) == False
+
+    @pytest.mark.parametrize(
+        "bbox1, bbox2",
+        [
+            bbox_covers_bbox,
+            bbox_contained_inside_bbox,
+            bbox_intersecting_bbox,
+            bbox_touches_edge_only_bbox,
+        ],
+    )
+    def test_geom_intersects(self, bbox1, bbox2):
+        assert bbox_intersects_bbox(bbox1, bbox2) or bbox_intersects_bbox(bbox2, bbox1) == True
```

Comment:
this looks strange. doesn't this allow either statement to fail? not sure about precedence of operators here, but one also shouldn't need to wonder when reading code. best to use 2 different statements or at least parentheses.